inc/class-s3-uploads-stream-wrapper.php

```diff
@@ -17,7 +17,34 @@ public static function register( Aws\S3\S3Client $client)
 		static::$client = $client;
 	}
 
+	// Override
+	public function stream_flush() {
+
+		// Add some HTTP headers which can be defined in wp-config.php
+
+		// Expires:
+		if ( defined( 'S3_UPLOADS_EXPIRES' ) ) {
+			$this->params[ 'Expires' ] = S3_UPLOADS_EXPIRES;
```

Comment:
I think we need a better name for the constant. On first glance, I thought this was about the object lifecycle on s3 - rather than the browser cache expires header (or maybe this is for object expires lifcycle? :confused: )